addStyles.js

```diff
@@ -143,6 +143,20 @@ function createLinkElement(options) {
 function addStyle(obj, options) {
 	var styleElement, update, remove;
 
+	// If a transformCssOnLoad function was defined, run it on the css
+	if (options.transformCssOnLoad && obj.css) {
```

Comment:
`transformCssOnLoad` sounds fine to me. `transform` alone does not communicate the intent.